docs/customization/model.rst

```diff
@@ -250,3 +250,167 @@ you'll need to update its form type. Check how to do it :doc:`here </customizati
     If you want the new field of your entity to be translatable, you need to extend the Translation class of your entity.
     In case of the ShippingMethod it would be the ``Sylius\Component\Shipping\Model\ShippingMethodTranslation``.
     Also the form on which you will add the new field should be the TranslationType.
+
+
+
+
+
+
+
+
```

Comment:
Okay, the above lines are redundant, and should be removed by you before we merge this PR :)